### DISCUSSION TITLE
Recent CVEs fixed in 3.3.x tree

### DISCUSSION THREAD

**Discussion by duckunix** [cobbler/cobbler#3109] [Version specific questions]
Due to many issues, we are currently stuck on Cobbler 2.8.5 on RHEL 7.  I just joined the team which is responsible for Cobbler within my company, and I have been tasked with doing a security review of our installation.

I see there were 3 CVE which were recently fixed in 3.3.x (CVE-2021-40323, CVE-2021-40324, and CVE-2021-40325). I am trying to figure out if these issues apply to 2.8.5.  Does anyone have any clue if these are something I need to worry about?

Thanks!

**Comment by SchoolGuy** [comment]
Yes all three affected issues are applying in short. It might be harder to exploit but they do apply. Since I know the migration can be painful I can try to assist on a best-effort basis on a migration. If you want you can contact me in private for that. A backport for those CVEs to a Python 2 version is definitely not on the table. If you require the web-interface I can try to prioritize the backports for 3.2.x which are already open as a PR.
